File: level 30/cw/cw3.py
#3) შექმენით ფუნქცია რომელშიც იქნება რიცხვებით სავსე სია ხოლო ფუნქცია დააბრუნებს ამ სიის ელემენტების ჯამს
def sum_of_list(numbers):
    """
    იღებს რიცხვებით სავსე სიას და აბრუნებს მისი ელემენტების ჯამს.
    :param numbers: რიცხვებით სავსე სია
    :return: სიის ელემენტების ჯამი
    """
    return sum(numbers)

# ფუნქციის გამოყენება
numbers_list = [3, 7, 12, 5, 10]  # რიცხვებით სავსე სია
result = sum_of_list(numbers_list)

print("სია:", numbers_list)
print("სიის ელემენტების ჯამი:", result)

File: level 71/cw/cw3.js
let numbers = [1, 2, 3, 4, 5, 6, 7, 8, 9, 10]; // რიცხვებით სავსე სია
let sumOfSquares = 0;

for (let i = 0; i < numbers.length; i++) {
    if (numbers[i] % 2 === 0) {  // ლუწი რიცხვის შემოწმება
        sumOfSquares += numbers[i] ** 2;  // ლუწი რიცხვის კვადრატი
    }
}

console.log("ლუწი რიცხვების კვადრატების ჯამი:", sumOfSquares);
